ui/page/channel/view.jsx

```diff
@@ -232,130 +251,137 @@ function ChannelPage(props: Props) {
 
   return (
     <Page className="channelPage-wrapper" noFooter>
-      <header className="channel-cover">
-        <div className="channel__quick-actions">
-          {isMyYouTubeChannel && (
-            <Button
-              button="alt"
-              label={__('Claim Your Channel')}
-              icon={ICONS.YOUTUBE}
-              navigate={`/$/${PAGES.CHANNELS}`}
+      <ChannelPageContext.Provider value>
+        <header className="channel-cover">
+          <div className="channel__quick-actions">
+            {isMyYouTubeChannel && (
+              <Button
+                button="alt"
+                label={__('Claim Your Channel')}
+                icon={ICONS.YOUTUBE}
+                navigate={`/$/${PAGES.CHANNELS}`}
+              />
+            )}
+            <JoinMembershipButton uri={uri} />
+            {!channelIsBlackListed && <ClaimShareButton uri={uri} webShareable />}
+            {!(isBlocked || isMuted) && <ClaimSupportButton uri={uri} />}
+            {!(isBlocked || isMuted) && (!channelIsBlackListed || isSubscribed) && (
+              <SubscribeButton uri={permanentUrl} />
+            )}
+            {/* TODO: add channel collections <ClaimCollectionAddButton uri={uri} fileAction /> */}
+            <ClaimMenuList uri={claim.permanent_url} inline isChannelPage />
+          </div>
```

Comment:
:small_red_triangle:  In mobile, the channel page's context menu (the one in the banner) no longer works.